source/exe/BUILD

```diff
@@ -81,7 +82,7 @@ envoy_cc_library(
             ":sigaction_lib",
             ":terminate_handler_lib",
         ],
-    }) + envoy_cc_platform_dep("platform_impl_lib"),
+    }) + envoy_cc_platform_dep("platform_impl_lib") + envoy_google_grpc_external_deps(),
```

Comment:
@jplevyak FYI, I'm adding this macro in this PR that you might also find useful.